ParisBNF/abb/BNFabb88.xml

```diff
@@ -0,0 +1,381 @@
+                                        quire structure has become invisible or lost.</note>
+                                </collation>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="2" writtenLines="18">
+                                    <ab type="pricking">Pricking is partly visible.</ab>
+                                    <ab type="ruling">Ruling is not visible.</ab>
+                                    <ab type="ruling" subtype="pattern" cert="low"
+                                        >1A-1A-1A1A/0-0/0-0/C</ab>
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <seg type="script">18th-century script</seg>
+                                <seg type="ink">Black, red.</seg>
+                                <desc>Careful, irregular; by a good scribe. <persName
+                                        ref="PRS1071dAbbadi"/> describes the handwriting as <foreign
+                                        xml:lang="fr">ʻguịlḥ moderneʼ</foreign>. </desc>
+                            </handNote>
+                        </handDesc>
+                        <decoDesc>
+                            <decoNote xml:id="d1" type="miniature">
+                                <locus target="#2v"/>
+                            </decoNote>
+                            <decoNote xml:id="d2" type="miniature">
+                                <locus target="#3v"/>
+                                <desc><ref type="authFile" corresp="AT1042HagiographicScene"
+                                            ><persName ref="PRS9848Walatta"/>Walatta Ṗeṭros taming
+                                        the hippopotamuses on Lake Tana.</ref></desc>
+                                <note>Interpretation of the miniature is taken from <bibl>
+                                        <ptr target="bm:BoscWion2010Abbadie"/>
+                                        <citedRange unit="page">90</citedRange>
+                                    </bibl>, containing a colorful reproduction of the miniature.
+                                </note>
+                            </decoNote>
+                            <decoNote xml:id="d3" type="miniature">
+                                <locus target="#4r"/>
+                            </decoNote>
+                        </decoDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus target="1ra"/>
+                                    <desc type="GuestText">
+                                        <title type="incomplete" ref="LIT7073TammeraW">Miracle of
```

Comment:
we need an ID for a single miracle

Reply:
i took a closer look, it is not a miracle, it is omitted part of the gadl!